src/ArcGISEarth.AutoAPI.Examples/ArcGISEarth.AutoAPI.Examples.csproj

```diff
@@ -104,6 +143,22 @@
     <Resource Include="examples.txt" />
```

Comment:
It's better to remove `examples.txt`

Reply:
I have removed the examples.txt and the whole resource file